### PR TITLE
Musark 940 - Adding an events grid for object

### PR DESCRIPTION
**Pull request by jarlah**
Må finne ut hvordan vi kan bytte ut flyttehistorikken.

### CODE REVIEW

File: src/models/event.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { parseISODate, DATE_FORMAT_DISPLAY } from '../shared/util';
+import MusitActor from './actor';
+import uniq from 'lodash/uniq';
+import { I18n } from 'react-i18nify';
+import concat from 'lodash/concat';
+import { Observable } from 'rxjs';
+
+export default class Event {}
+
+Event.concatAnalysesWithMoves = (ajaxGet = simpleGet, ajaxPost = simplePost) => (val) =>
```
Comment (steinaol) [inline src/models/event.js]: Should this have another name:  analysesAndMovesForObjects, or something like that?

File: src/models/event.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  Observable.forkJoin(
+    Analyses.getAnalysesForObject(ajaxGet)(val),
+    MusitObject.getLocationHistory(ajaxGet, ajaxPost)(val).map(locations =>
```
Comment (steinaol) [inline src/models/event.js]: Does not correspond with definition in Object model?

File: src/models/object.js [hunk 3]
```diff
@@ -86,7 +87,7 @@ MusitObject.moveObject = (ajaxPut = simplePut) => (
   return ajaxPut(`${Config.magasin.urls.api.storagefacility.baseUrl(museumId)}/moveObject`, data, token, callback);
 };
 
-MusitObject.getLocationHistory = (ajaxGet = simpleGet, ajaxPost = simplePost) => ({ objectId, museumId, token, callback }) => {
+MusitObject.getLocationHistory = (ajaxGet = simpleGet) => ({ objectId, museumId, token, callback }) => {
```
Comment (steinaol) [inline src/models/object.js:90]: See comment in objects.js

File: src/modules/events/EventsComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+const Table = require('reactable').Table;
+const Tr = require('reactable').Tr;
+const Td = require('reactable').Td;
```
Comment (steinaol) [inline src/modules/events/EventsComponent.jsx]: Couldn't we use 
import { Table, Tr, Td } from 'reactable';
?

**Review by steinaol — COMMENTED** [review]
See comments

File: src/models/event.js (hunk 2 above)
Comment (jarlah) [reply src/models/event.js]: :+1:

File: src/models/object.js (hunk 3 above)
Comment (jarlah) [reply src/models/object.js:90]: :+1:

File: src/models/event.js (hunk 1 above)
Comment (jarlah) [reply src/models/event.js]: Event.getAnalysesAndMoves :+1:

File: src/modules/events/EventsComponent.jsx (hunk 4 above)
Comment (jarlah) [reply src/modules/events/EventsComponent.jsx]: :+1:

**Review by steinaol — APPROVED** [review]
Looks good to me

**Pull request merged**